### DISCUSSION TITLE
PKHeX save file size workaround

### DISCUSSION THREAD

**Discussion by LaReunion974** [kwsch/PKHeX#3998] [Show and tell]
I found a workaround for this: 

1. have the game + the last update and/or dlc
2. start a new game to generate a new save file 
3. immediately save your game (the save file should be 3161ko)
4. stop the game 
5. apply the latest update (2.0.1) and/or dlc
6. start the game with the same save file you created 
7. make a MANUAL save 
8. stop your game again 
9. check your save file directory (should be 4300ko)
10. launch PKHeX and select your save file and you should be good to edit your save👍

**Comment by sora10pls** [comment]
The idea is not to create workarounds; users can share their save data so that the developers can add support for those unrecognized file sizes.

In the end, it's better to be patient and wait for the program to receive updates that accommodate the unknown save files.

**Reply by Doctor-Pokemon** [reply]
I am very interested in the idea of sending in my savegames to help with any research necessary.
Should I message Kurt on ProjectPokemon.org or is there an official channel?
How should I best go about this?

**Comment by LokeYourC3PH** [comment]
Having the issue with the save file size of Violet being at 4401596 bytes (latest update):
![image](https://github.com/kwsch/PKHeX/assets/5807595/48e0bb71-c836-450b-ac7a-0ccc619d9f5b)
